src/native-automation/request-pipeline/index.ts

```diff
@@ -358,12 +358,18 @@ export default class NativeAutomationRequestPipeline extends NativeAutomationApi
     }
 
     private _getUploadPostData (event: Protocol.Fetch.RequestPausedEvent): string | undefined {
-        if (!event.request.postData)
+        //NOTE: https://github.com/DevExpress/testcafe/issues/8198
```

Comment:
Please remove this comment. Your PR already has the link to the initial issue, so it's easy to find this issue by breadcrumbs.